src/main/java/net/mcreator/blockly/java/blocks/CustomDependencyBlock.java

```diff
@@ -49,6 +49,8 @@ public CustomDependencyBlock() {
 				master.append("/*@ItemStack*/");
 			else if (deptype.equalsIgnoreCase("blockstate"))
 				master.append("/*@BlockState*/");
+			else if (deptype.equalsIgnoreCase("damagesource"))

```

Comment:
The shortcut doesn't work because the dependency is named `source` and not `damagesource`